syntax = "proto3";
package tinkoff.proto;

// Request
message AcceptDealRequest { string deal_id = 1; }
